public/app/plugins/panel/gettingstarted/GettingStarted.tsx

```diff
@@ -107,9 +109,9 @@ export class GettingStarted extends PureComponent<PanelProps, State> {
             </div>
             {currentStep < steps.length - 1 && (
               <Button
-                className={cx(styles.backForwardButtons, styles.forward)}
+                className={styles.backForwardButtons}
                 onClick={this.onForwardClick}
```

Comment:
Is it worth putting interactions here to see if anyone even sees this button? It took me months to realize there was a button there